src/js/components/FormField/FormField.js

```diff
@@ -169,16 +168,8 @@ class FormFieldContent extends Component {
         border.position === 'outer' &&
         (border.side === 'all' || border.side === 'horizontal' || !border.side);
       if (abut) {
-        // marginBottom is set to overlap adjacent fields
```

Comment:
This is also a breaking change. We need to handle the case where the border is on all sides, and the focus indicator is visible appropriately, when there is no margin between the fields.

Reply:
so this change needs to get reverted @joybhallaa